tests/integration/suite/conftest.py

```diff
@@ -20,6 +20,8 @@
     create_cluster, \
     import_cluster
 
+# This is where all charts for testing are
+CATALOG_URL = "https://github.com/rancher/integration-test-charts"
```

Comment:
This already exists in this file https://github.com/rancher/rancher/blob/master/tests/integration/suite/conftest.py#L46